### ISSUE TITLE
Improve appearance of settings tab

### ISSUE THREAD

**Issue by Coco555-cyber** [uchicago-cs/chigame#919]
The settings tab currently overlaps jarringly with the navbar, and also does not have a transition animation at all. The accessibility button is also the default appearance, rather than the buttons present on the rest of the page.

**Comment by abargher** [comment]
Issue Score: Excellent

Comments:
Nice work!
Here are some ways this could be improved:
- Even though you can add labels to issues to categorize them, you should avoid titles that could plausibly apply to multiple teams.
